### PR TITLE
Modifications for 2015-08-11

### PR DESCRIPTION
**Pull request by marians**
As described in a [blog post](https://blog.giantswarm.io/better-way-to-define-services/) already, the model of how things are run on Giant Swarm changes a little. These modifications in our legal documents reflect these changes.

For review @teemow @othylmann

### CODE REVIEW

File: limits.md [hunk 1]
```diff
@@ -2,16 +2,15 @@
...
 * an unlimited number of components per service
 
 However, we enforce these restrictions:
 
-* any application older than 7 days (from creation time) will be deleted
+* any service older than 7 days (from creation time) will be deleted
   * except if you have configured any [custom domain name](https://docs.giantswarm.io/reference/swarm-json/#own-domain-name)
-* a maximum of 10 instances (containers) per component
-* a memory (RAM) limit of 512 MB per instance (container)
+* a maximum of 10 component instances (containers) per component
```
Comment (teemow) [inline limits.md]: I'd still say instances here. no need to call them component instances.

Comment (marians) [inline limits.md]: Just to make clear in this context what they are? We also use the term "component instances" as a long form in a few places, to make very clear that instances are actually component instances.

Comment (teemow) [inline limits.md]: It's `component instance per component`. I don't see a need to confuse people with this. I would ask myself if there are other instances.

**Comment by marians** [comment]
Changed "component instances" to "instances"

**Pull request merged**